draft-gruessing-moq-requirements.md

```diff
@@ -201,14 +218,20 @@ place.
 |**Senders/Receivers**|  One to Many
 |**Bi-directional**| No
 
-Where media is received from a live broadcast or stream. This may comprise of
-multiple audio or video outputs with different codecs or bitrates. This may also
-include other types of media essence such as subtitles or timing signalling
-information (e.g. markers to indicate change of behaviour in client such as
-advertisement breaks). The use of "live rewind" where a window of media behind
-the live edge can be made available for clients to playback, either because the
-local player falls behind edge or because the viewer wishes to play back from a
-point in the past.
+Where media is received from a live broadcast or stream either as a broadcast
+with fixed duration or as ongoing 24/7 output. The number of receivers may vary
```

Comment:
Perhaps "Where media is received from a live broadcast or stream either as a broadcast with fixed duration or **for an unknown duration, potentially** as ongoing 24/7 output.